build.gradle

```
@@ -173,6 +173,20 @@ configurations {
     }
 }
 
+def findJarByName(Configuration config, String name) {
+    config.find { File file -> file.name.startsWith(name)}
+}
+
+task setupGatkLaunch(type: Copy) {
+    def gatkJar = { findJarByName(configurations.compile, 'gatk') }
+    from {
+        zipTree( gatkJar ).matching{ include 'gatk-launch'}
+    }
```

Will this silently continue if there's no `gatk-launch` inside the jar?